gen3.datacommons.io/manifest.json

```diff
@@ -359,7 +357,7 @@
       "strategy": "auto",
       "min": 1,
       "max": 2,
```

Comment:
should we also remove or update all the scaling sections to only deploy 1 pod for each service?

Reply:
I think I'll scope that in for a future change to discuss with PE. Thanks for this, Pauline!